Admin/FieldDescription.php

```diff
@@ -16,7 +16,7 @@
 class FieldDescription extends BaseFieldDescription
 {
     /**
-     * {@inheritdoc}
+     * Constructor.
```

Comment:
**Add PHPDoc blocks for all classes, methods, and functions;**
http://symfony.com/doc/current/contributing/code/standards.html#documentation

IMO *all* means really all, there is no exception.